packages/orbit-components/src/utils/common/index.ts

```diff
@@ -1,9 +1,34 @@
 import type { FlattenSimpleInterpolation } from "styled-components";
 import { css } from "styled-components";
 
+import type { ObjectProperty } from "../../common/types";
+
 export const firstToUpper = (str: string): string => str.charAt(0).toUpperCase() + str.slice(1);
 
 export const defaultFocus = (): FlattenSimpleInterpolation => css`
   outline: 2px solid Highlight;
   outline: 2px solid -webkit-focus-ring-color;
 `;
+
+const setValue = (value: string | number): string =>
+  typeof value === "number" ? `${value}px` : value;
```

Comment:
```suggestion
  typeof value === "number" && !== 0 ? `${value}px` : value;
```

Because if it is 0, it should just be 0 💅